app/src/androidTest/java/com/github/multimatum_team/multimatum/util/MockGroupRepository.kt

```diff
@@ -56,6 +56,14 @@ class MockGroupRepository(initialContents: List<UserGroup>) : GroupRepository()
         groups.remove(id)
     }
 
+    override suspend fun rename(id: GroupID, newName: String) {
+        groups[id] = groups[id]!!.copy(name = newName)
```

Comment:
`rename` is public, so it should check its arguments and in particular throw an `IllegalArgumentException` if `groups[id] == null`, not a `NullPointerException`. So I think that a `require` is needed here.

Reply:
This class is only used in tests but sure